src/sentry/auth/authenticators/u2f.py

```diff
@@ -99,9 +103,25 @@ def try_enroll(self, enrollment_data, response_data, device_name=None):
 
     def activate(self, request):
         challenge = dict(u2f.begin_authentication(self.u2f_app_id, self.get_u2f_devices()))
+
+        # TODO for completeness change to webauthn when functionalities for everything else is done
+        # server = U2FFido2Server(
```

Comment:
i would remove the blocks of code for now -- usually not ideal to commit commented code out to master.